app/bundles/EmailBundle/Model/EmailModel.php

```diff
@@ -154,7 +165,9 @@ public function __construct(
         SendEmailToContact $sendModel,
         DeviceTracker $deviceTracker,
         RedirectRepository $redirectRepository,
-        CacheStorageHelper $cacheStorageHelper
+        CacheStorageHelper $cacheStorageHelper,
+        ContactTracker $contactTracker,
+        \Mautic\LeadBundle\Model\DoNotContact $doNotContact
```

Comment:
Let's add the namespace to the use statement at the top of the file. Reasons:
1. Shorter.
2. More readable.
3. Consistency.